about.html

```diff
@@ -0,0 +1,181 @@
+            foundations, lawyers, and activists - we hope you'll consider
+            joining us this year to build a stronger, more vibrant commons
+            together
+          </p>
+          <p class="about-contact">
+            Please contact us per Email for nay further questions about CC
+            Global Summit 2015!
+          </p>
+          <p class="about-email">
+            uld@dmgmedia.com
+            <i class="fa-solid fa-regular fa-chevron-right"></i>
+          </p>
+        </div>
+      </div>
+    </section>
+    <section class="about-2" id="about-2">
+      <h2>Urban Landscape and Design Expo 2023 logo</h2>
+      <hr class="about-hr" />
+      <p class="about-2-p-1">
+        The logo of Urban Landscpae and Design 2023 was decided through the logo
+        competition from 8, June to 7, July.
+      </p>
+      <p class="about-2-p-2">
+        The logo of CC Global Summit 2015 was decided through the logo
+        competition from 8, June to 7, July. After online public voting and
+        Selection Committee’s judging process, the logo from Naresh Agrawal won
+        the prize. After consultation with Mr. Agrawal, we optimized the
+        prize-winning-logo for this year’s summit as follows:
+      </p>
+      <img src="./assests/udle-2022-main.webp" alt="desk-about" />
+      <hr class="logo-hr" />
+      <div class="past-events">
+        <h2>See the past Urban Landscape and Design Expo</h2>
+        <hr class="about-hr" />
+        <p>
+          Take a look at the past Urban Landscape and Desing Expo which took
+          place in Dubai, United Arab Emirates.
+        </p>
+      </div>
+    </section>
+    <div class="cards-section">
+      <div class="past-card past-image-1">
+        <h1>2021</h1>
+        <p>Urban Landscape and Desgin Expo in Dubai</p>
+      </div>
+      <div class="past-card past-image-2">
+        <h1>2022</h1>
+        <p>Urban Landscape and Desgin Expo in Dubai</p>
+      </div>
+    </div>
+    <!-- about section starts here -->
+
+    <!-- partners section starts here  -->
+    <section class="partner-2" id="partner">
+      <h1>Partners</h1>
+      <hr class="about-hr" />
+      <div class="partner-images">
+        <img src="./assests/mozilla.png" alt="mozilla" />
+        <img src="./assests/google.png" alt="google" />
+        <img src="./assests/naver.png" alt="naver" />
+        <img src="./assests/daumkakao.png" alt="daumkakao" />
+        <img src="./assests/airbnb.png" alt="airbnb" />
+      </div>
+    </section>
+    <!-- partners section ends here  -->
+
+    <!-- footer starts here  -->
+    <footer class="footer-2">
+      <div class="cc-2">
+        <img src="./assests/cc.png" alt="creative-common" />
+        <img src="./assests/cc white.png" alt="creative-common" />
+        <p>
+          creative <br />
+          commons Dubai
+        </p>
+      </div>
+      <span>2023 Creative Commons Dubai. Some Rights Reserved.</span>
+    </footer>
```

Comment:
- [ ] Great work removing the Partner section from the _**About**_ page. As highlighted by the previous reviewer, the design of the footer for the desktop version is not identical to that of the Behance template. In the _**Behance**_ template, the footer occupies the bottom of the page with no margin or padding at the bottom but yours has margin at the bottom. Please remove the `margin-bottom` of `50px` that you declared in the `footer-2` selector. 😎 💻 

### Your Design

![image](https://github.com/Kaiserabbas/Capstone/assets/77106962/8fd8499d-fc05-4cae-b570-5cada6241aba)

### Behance Design

![image](https://github.com/Kaiserabbas/Capstone/assets/77106962/669b5855-552c-44b4-9af0-50863a202fb5)